ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -24,9 +24,9 @@ let with_lock = Xapi_stdext_threads.Threadext.Mutex.execute
 
 let finally = Xapi_stdext_pervasives.Pervasiveext.finally
 
-module DD = Debug.Make (struct let name = "xapi_vm_migrate" end)
+module Impl = Debug.Make (struct let name = "storage_impl" end)
```

Comment:
Is this the right name? By opening the module below, its name becomes irrelevant. Is this not logging errors for `xapi_vm_migrate`? In that case `struct let name = __MODULE__ end` would work. The name `Impl` doesn't make sense to me. This is about debug logging; why is `Impl` better than `DD` from before?

Reply:
This seems to have been auto generated, and the build did not like having the `module DD ...` included. I am not sure on this.

Reply:
This is not right... please undo the change.